packages/blockchain/flow/src/lib/flow.sdk.spec.ts

```diff
@@ -0,0 +1,265 @@
+import { TatumFlowSDK } from './flow.sdk'
+import {
+  BlockchainTestData,
+  REPLACE_ME_WITH_TATUM_API_KEY,
+  TEST_DATA,
+  walletTestFactory,
+} from '@tatumio/shared-testing'
+import { SdkWithWalletFunctions } from '@tatumio/shared-blockchain-abstract'
+import {
+  ChainFlowBurnNft,
+  ChainFlowMintMultipleNft,
+  ChainFlowMintNft,
+  ChainFlowTransferNft,
+  Currency,
+  TransferFlow,
+  TransferFlowCustomTx,
+} from '@tatumio/shared-core'
+import dedent from 'dedent-js'
+
+describe('TatumFlowSDK', () => {
+  jest.setTimeout(999999)
```

Comment:
why is it here?

Reply:
Test were failing on timeout so I prolonged it.